pages/common/git-stripspace.md

```diff
@@ -0,0 +1,16 @@
+# git stripspace
+
+> Read text, such as commit messages, notes, tags and branch descriptions, from the standard input and clean it in the manner used by Git.
+> More information: <https://git-scm.com/docs/git-stripspace>.
+
+- Trim whitespace from a file:
+
+`cat {{path/to/file}} | git stripspace`
+
+- Trim whitespace and git comments from a file:
```

Comment:
`git` or `Git`?

Reply:
I think `Git`, actually.  Any opinions, anyone?

Reply:
`Git` is preferred in the man pages I think, unless it's the name of the `git` command.